tests/lib/prepare.sh

```diff
@@ -152,6 +152,10 @@ EOF
     if [ "$REMOTE_STORE" = staging ]; then
         # shellcheck source=tests/lib/store.sh
         . "$TESTSLIB/store.sh"
+        # reset seeding data that is likely tainted with production keys
+        systemctl stop snapd.service snapd.socket
```

Comment:
what starts it again?

Reply:
setup_staging_store itself (it stops again too but that's idempotent)